src/test/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/xsv/LocatableXsvFuncotationFactoryUnitTest.java

```diff
@@ -121,6 +126,13 @@ private Object[][] provideForTestGetName() {
 
     @DataProvider
     private Object[][] provideForTestCreateFuncotations() {
+        Object[][] arr1 = provideDataForTestCreateFuncotations(false);
+        Object[][] arr2 = provideDataForTestCreateFuncotations(true);
+        return Stream.concat(Arrays.stream(arr1), Arrays.stream(arr2)).
+                toArray(size -> (Object[][]) Array.newInstance(arr1.getClass().getComponentType(), arr1.length + arr2.length));
```

Comment:
Could you use the preexisting utility method `Utils.concat(arr1, arr2, Object[][]::new)` here?

Reply:
ok, done